espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/model/ElementValueParams.kt

```diff
@@ -1,5 +1,5 @@
 package io.appium.espressoserver.lib.model
 
 data class ElementValueParams(
```

Comment:
w3c also requires `text` parameter to be set, which contains a string rather than array

Reply:
Okay. Will add text and update routes, too

https://github.com/appium/appium-base-driver/blob/master/lib/protocol/routes.js